wrangling.Rmd

```diff
@@ -42,35 +40,34 @@ By the end of the chapter, readers will be able to:
       - `|`
       - `|>` and `%>%`
 
-## Data frames and vectors
+## Data frames, vectors, and lists
 
-We now know how to load data into R from various file formats. 
-Given that most of the tools we have used to import data represent the data 
-as a *data frame* in R, 
-we will spend some time learning more about data frames 
-to deepen our understanding of them. 
-This will be useful prerequisite knowledge to use
-and manipulate these objects in our data analysis.
+In Chapters \@ref(intro) and \@ref(reading), *data frames* were the focus:
+we learned how to import data into R as a data frame, and perform basic operations on data frames in R.
+In the remainder of this book, this pattern continues. The vast majority of tools we use will require 
+that data are represented as a data frame in R. Therefore, in this section,
+we will dig more deeply into what data frames are and how they are represented in R.
+This knowledge will helpful in effectively utilizing these objects in our data analyses.
 
 ### What is a data frame?
 
-A **data frame** is a table-like structure 
-and is a very common object for storing data in R. 
-We have already seen and worked with data frames in Chapter \@ref(reading), 
-and here we will get to know them in more detail.
+A data frame \index{data frame!definition} is a table-like structure for storing data in R.  Data frames are
+important to learn about because most data that you will encounter in practice
+can be naturally stored as a table.  In order to define data frames precisely,
+we need to introduce a few technical terms:
 
-To help prepare us to do that, 
-we need to introduce a few commonly used terms in statistics:
+- **variable:** a \index{variable} characteristic, number, or quantity that can be measured.
+- **observation:** all \index{observation} of the measurements for a given entity.
+- **value:** a \index{value} single measurement of a single variable for a given entity.
 
-  - **observation** - all of the quantities and/or qualities we collect from a
-    given entity/object. An example of an entity/object is a particular person.
-  - **variable** - any characteristic, number, or quantity that can be measured
-    or collected (*e.g.,* adult height)
-  - **value** - a single collected quantity or a quality from a given
-    entity/object (*e.g.,* the adult height of a particular person)
-
-From a data perspective, a data frame is a rectangle where the rows are the
-observations and the columns are the variables:
+Given these definitions, a **data frame** is a table of values where the rows
```

Comment:
This is a very good point, but I'm feeling a bit brain dead right now and don't have a great way to fix it. I'll open an issue and tag it as a bug so that we have to address it before v1.0 comes out.